src/api/api.ts

```diff
@@ -17,6 +18,48 @@ export function saveEncounter(abortController: AbortController, payload, encount
   });
 }
 
+export function saveAttachment(patientUuid, content, conceptUuid, date, encounterUUID, abortController) {
+  const url = '/ws/rest/v1/attachment';
+
+  const formData = new FormData();
+
+  const cameraUploadType = content
+    ?.split(';')[0]
+    .split(':')[1]
+    .split('/')[1];
+
+  const fileCaption = typeof content === 'object' ? content.name : `camera-upload.${cameraUploadType}`;
+
+  formData.append('fileCaption', fileCaption);
+  formData.append('patient', patientUuid);
+
+  if (typeof content === 'object') {
+    formData.append('file', content);
+  } else {
+    formData.append('file', new File([''], 'OHRIFileConceptTest.png'), 'OHRIFileConceptTest.png');
```

Comment:
Hi @arodidev is the file name always going to be `OHRIFileConceptTest.png`? I think it should be something more dynamic

Reply:
Noted, will resolve